index.html

```diff
@@ -44,129 +65,150 @@ <h4 class="date">Published on Feb 27, 2014</h4>
+                <p>
             </aside>
 
             <aside>
                 <img src="images/side-image-05.jpg">
-                <h3>GitHub - Why Microsoft Paid $7.5B for the Future of Software! - A Case</h3>
-                <small>Valuetainment</small>
-                <small>444,985 views</small>
+                <p><strong>GitHub - Why Microsoft Paid $7.5B for the Future of Software! - A Case</strong><br>
+                    <span class="text-font"><small>Valuetainment<br>
+                    444,985 views</small></span>
+                <p>
             </aside>
 
             <aside>
                 <img src="images/side-image-06.jpg">
-                <h3>How to create a 3D Terrain with Google Maps and height maps in Photoshop</h3>
-                <small>Orange Box Ceo</small>
-                <small>29,296 views</small>
+                <p><strong>How to create a 3D Terrain with Google Maps and height maps in Photoshop</strong><br>
+                    <span class="text-font"><small>Orange Box Ceo<br>
+                    29,985 views </small></span>
+                <p>
             </aside>
 
             <aside>
                 <img src="images/side-image-07.jpg">
-                <h3>How to create a 3D Terrain with Google Maps and height maps in Photoshop</h3>
-                <small>deeplizard</small>
-                <small>329,285 views</small>
+                <p><strong>How to create a 3D Terrain with Google Maps and height maps in Photoshop</strong><br>
+                    <span class="text-font"><small>deeplizard<br>
+                    329,285 views </small></span>
+                <p>
             </aside>
 
             <aside>
                 <img src="images/side-image-08.jpg">
-                <h3>Top 10 Linux Job Interview Questions</h3>
-                <small>Computerphile</small>
-                <small>2,129,277 views</small>
+                <p><strong>Top 10 Linux Job Interview Question</strong><br>
+                    <span class="text-font"><small>Computerphile<br>
+                        2,129,277 views </small></span>
+                <p>
             </aside>
 
             <aside>
                 <img src="images/side-image-09.jpg">
-                <h3>Running an SQL Injection Attack - Computerphile</h3>
-                <small>Computererphile</small>
-                <small>2,217,477 views</small>
+                <p><strong>Running an SQL Injection Attack - Computerphile</strong><br>
+                    <span class="text-font"><small>Computererphile<br>
+                    2,029,985 views </small></span>
+                <p>
             </aside>
 
             <aside>
                 <img src="images/side-image-01.jpg">
-                <h3>Contributing to Open Source Part II: The Real Way</h3>
-                <small>The Odin Project</small>
-                <small>29,985 views</small>
+                <p><strong>Contributing to Open Source Part II: The Real Way</strong><br>
```

Comment:
Don't use the <strong> tag to make regular text bold, use CSS instead. [The HTML Strong Importance Element (<strong>) indicates that its contents have strong importance, seriousness, or urgency. Browsers typically render the contents in bold type.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong)